azurerm/resource_arm_storage_blob.go

```diff
@@ -201,6 +209,13 @@ func resourceArmStorageBlobCreate(d *schema.ResourceData, meta interface{}) erro
 		}
 	}
 
+	blob.Metadata = expandBlobMetadata(d)
```

Comment:
As these functions are package wide could we rename this to
```suggestion
	blob.Metadata = expandStorageAccountBlobMetadata(d)
```